include/vcpkg/base/messages.h

```diff
@@ -1011,6 +1063,10 @@ namespace vcpkg
                     (msg::value),
                     "{value} is a sha.",
                     "SHA512's must be 128 hex characters: {value}");
+    DECLARE_MESSAGE(IncompatibleOptionSetting,
+                    (msg::value, msg::option),
+                    "{value} is the 'main' option.",
+                    "--{value} is only valid with --{option}.");
```

Comment:
```suggestion
    DECLARE_MESSAGE(MutuallyRequiredOption,
                    (msg::value, msg::option),
                    "{value} is a second {option} switch",
                    "--{value} must be used with --{option}.");
```

which leaves room for this one either here or in a separate review:

```
    DECLARE_MESSAGE(MutuallyExclusiveOption,
                    (msg::value, msg::option),
                    "{value} is a second {option} switch",
                    "--{value} can not be used with --{option}.");
```

Should we include the -- in the `{value}`? That way we can say exactly what the user typed if we are accepting some other form, although that may be at most a theoretical concern.

As an aside, should we always say "switch" even though internally we distinguish between switch/option/multi-option?